ckanext/scheming/tests/test_validation.py

```diff
@@ -52,6 +53,8 @@ def test_choice_field_accepts_valid_choice(self):
             )
         assert_equals(d['category'], 'f2hybrid')
 
+        reset_db()
```

Comment:
Hmm. Maybe we can move this to to a class based on FunctionalTestBase instead (IIRC reset_db doesn't clean up everything)